static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -79,6 +79,13 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
           <span style={{gridArea: 'time'}}>
             <TimeSince date={feedbackItem.timestamp} />
           </span>
+          {feedbackItem.hasSeen ? null : (
+            <span
+              style={{gridArea: 'unread', display: 'inline-flex', alignItems: 'center'}}
```

Comment:
might be this `inline-flex` that makes center alignment not work quite right.

Reply:
oh i just added `justifyContent: center` and it works